_sips/minutes/2020-03-11-minutes.md

```diff
@@ -0,0 +1,216 @@
+---
+layout: sips
+title: SIP Meeting Minutes - March 11 2020
+
+partof: minutes
+---
+
+# Minutes
+
+The meeting took place with the following agenda:
+
+1. Quick setup, reviewing expectations for the retreat
+2. Review a list of "easy" dotty features/changes (see section below)
+3. Review enums
+4. Review creator applications
+5. Review `@infix` and `@alpha`
+6. Review dependent and polymorphic function types
+
+## Date and Location
+
+The meeting took place on the 11th March 2020 throughout the day at EPFL in Lausanne, Switzerland, and Zoom.
+
+The meeting wasn't broadcast.
+
+Minutes were taken by Dale Wijnand.
+
+## Committee Attendees
+
+* Martin Odersky ([@odersky](https://github.com/odersky)), EPFL
+* Sébastien Doeraene ([@sjrd](https://github.com/sjrd)), Scala Center
+* Guillaume Martres ([@smarter](https://github.com/smarter)), EPFL
+* Lukas Rytz ([@lrytz](https://twitter.com/lrytz)), Lightbend
+* Seth Tisue ([@SethTisue](https://github.com/SethTisue)), Lightbend
+
+## Sitting in
+
+* Adriaan Moors ([@adriaanm](https://github.com/adriaanm)), Lightbend
+* Nicolas Stucki ([@nicolasstucki](https://github.com/nicolasstucki)), EPFL
+* Jamie Thompson ([@bishabosha](https://github.com/bishabosha)), Scala Center
+* Darja Jovanovic ([@darjutak](https://github.com/darjutak)), Process Lead
+* Dale Wijnand ([@dwijnand](https://twitter.com/dwijnand)), secretary
+
+## Not present
+
+* Iulian Dragos ([@dragos](https://github.com/dragos)), Triplequote
+* Josh Suereth ([@jsuereth](https://github.com/jsuereth)), independent
+* Miles Sabin ([@milessabin](https://github.com/milessabin)), independent
+* Heather Miller ([@heathermiller](https://github.com/heathermiller)), CMU
+
+## Proceedings
+
+### Expectations
+
+* Martin: The plan is still to release Scala 3 this year, at the end of December.
+* Martin: However, the 'sterialized' TASTy won't be until 3.1, to a 3.0 promise.
```

Comment:
"stabilized", I think (it's me who heard it wrong). apparently I have virus precautions on the brain